libs/seo/state/src/state.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { EffectsModule } from '@ngrx/effects';
+
+import { DaffSeoTitleEffects } from './effects/title.effects';
+
+@NgModule({
+  imports: [
+    EffectsModule.forFeature([
+      DaffSeoTitleEffects,
+    ]),
+  ],
+})
+export class DaffSeoStateModule {}
```

Comment:
Does this need to be in the API?

Reply:
I suppose not, apps can simply load the effects themselves

Reply:
should I remove it?

Reply:
oh nvm I see what you mean, yeah the export was in a different commit, I'll add it